File: Programmers/Hash/위장.cpp
#include string
#include vector
#include map
#include iostream
using namespace std;

int solution(vectorvectorstring clothes) {
    int answer = 1;
    mapstring,int m;
    for(int i =0; i  clothes.size(); i++)
    {
        m[clothes[i][1]] += 1;
    }
    for(int i =0; i  clothes.size(); i++)
    {
        answer = (m[clothes[i][1]] + 1);
        m[clothes[i][1]] = 0;
    }
    return answer - 1;
}

File: C&Python/Hash/위장.py
def solution(clothes):
    answer = 1
    cnt = {}
    for i in range(len(clothes)):
        if clothes[i][1] not in cnt:
            cnt[clothes[i][1]] = 2
        else:
            cnt[clothes[i][1]] += 1
    #print(cnt)
    
    cnt_list = list(cnt.values())
    for i in cnt_list:
        answer *= i
        
    #print(cnt_list)
    
    return answer - 1